### PR TITLE
A couple of fixes for update control map wait logic.

### PR DESCRIPTION
**Pull request by kacf**
See individual commits.

### CODE REVIEW

**Comment by kacf** [comment]
@mender-test-bot start pipeline

**Review by lluiscampos — APPROVED** [review]
They look good to me

File: app/state.go [hunk 1]
```diff
@@ -2075,6 +2075,12 @@ func (c *controlMapPauseState) Handle(ctx *StateContext, controller Controller)
 
 	updateMapFromServerIn := time.Until(nextServerMapRefresh)
 
+	// For rapid feedback, use the smallest number of the map refresh time
+	// and the update poll interval. The latter is often much shorter.
+	if updateMapFromServerIn > controller.GetUpdatePollInterval() {
```
Comment (oleorhagen) [inline app/state.go:2080]: I think we have a Min function somewhere? :thinking:

**Review by oleorhagen — APPROVED** [review]

File: app/state.go (hunk 1 above)
Comment (kacf) [reply app/state.go:2080]: Yes, but for ints. Golang is really picky about types, so here it made more sense to just a do a check.

**Pull request merged**